ta/pkcs11/src/pkcs11_attributes.c

```diff
@@ -902,6 +922,20 @@ bool object_is_private(struct obj_attrs *head)
 	return false;
 }
 
+bool object_is_token(struct obj_attrs *head)
+{
+	return get_bool(head, PKCS11_CKA_TOKEN);
+}
+
+bool object_is_modifiable(struct obj_attrs *head)
+{
+	return get_bool(head, PKCS11_CKA_MODIFIABLE);
+}
+
+bool object_is_copyable(struct obj_attrs *head)
+{
+	return get_bool(head, PKCS11_CKA_COPYABLE);
+}
```

Comment:
maybe we can move these to **pkcs11_attributes.h** as inline functions.
if not, just add an empty line.

Reply:
I will add an empty line. We will also need to move get_bool if we want to make it an inline function

Reply:
ok as is, yet we could make these inline functins even if `get_pool()` isn't itself an inline function.